### DISCUSSION TITLE
New Docker Release - with server-side support

### DISCUSSION THREAD

**Discussion by ShortCipher5** [toeverything/AFFiNE#461] [General]
The Docker versions of AFFiNE before today were limited to a client implementation.

We heard your feedback loud and clear (you can always add more and discuss here: [https://feedback.affine.pro](https://feedback.affine.pro)).

So that's why we've released a new Docker version, it uses the `nightly-server-latest` tag.

In this version you can experience live multi-user collaborative editing and server-side storage support.

We are still ironing out the offline editing and sync ability, but expected that to be functioning and available soon too.

If you give this new Docker version a try, please do let us know about your experience.

**Comment by Jengro777** [comment]
I've moved some data to the unstable Affine, although it's not stable yet. But I know it will soon be my main note-taking and collaboration place.
I'm relieved to see that features for mind maps and math formulas are already planned in the roadmap.

Here, I hope Affine has the ability to visualize charts in the foreseeable future (surprise not a must)

![image](https://user-images.githubusercontent.com/35259040/192726305-0c4b139b-23d5-40f1-9cfe-b37d3bfe46f3.png)
